tests/resolvers/Query/verifyRole.spec.ts

```
@@ -0,0 +1,263 @@
+      expect(result).toEqual({
+        role: "",
+        isAuthorized: false,
+        error: "Authentication failed",
+      });
+    } else {
+      throw new Error("verifyRole is undefined");
+    }
+  });
+
+  test("should handle missing ACCESS_TOKEN_SECRET", async () => {
+    delete process.env.ACCESS_TOKEN_SECRET;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)